### ISSUE TITLE
Extra escaped quotes in -t command option when debugging

### ISSUE THREAD

**Issue by gmartinerro** [nithinpeter/jest-run-it#10]
The debugging command does not work because the `-t` option adds up a pair of escaped quotes for the test name. So instead of launching:

`node_modules/jest/bin/jest.js mytestFile.test.js -t "mytest" -c ....`

the debug command is launched using this:

`node_modules/jest/bin/jest.js mytestFile.test.js -t "\'mytest\'" -c ....`

When the test is defined this way:

` test('mytest', () =>{
   ...
   })
`

And the test is skipped.

**Comment by Langtonm** [comment]
I have the same issue. It looks like "run" generates the correct jest command but "debug" does not.
I think line 32 in commands.ts : -
```
  if (testName) {
    args.push('-t', quoteTestName(testName));
  }

```
should be : -
```
    if (testName) {
        args.push('-t', testName);
    }
```

Thanks for the wonderful plugin BTW, its very useful.

@gmartinerro - ITMT you can hack your installed version e.g. (vespa-dev-works.jestrunit-0.3.3/out/commands.ts) in : -
Windows %USERPROFILE%\.vscode\extensions
macOS ~/.vscode/extensions
Linux ~/.vscode/extensions

**Comment by nithinpeter** [comment]
This should be fixed in the latest version (v4.0.0). Thanks for reporting the issue.
